### PR TITLE
allow listing files and file downloads for any service

### PR DESCRIPTION
**Pull request by jault3**
new commands added are `file list` and `file download`

### CODE REVIEW

File: Docs.md [hunk 1]
```diff
@@ -324,6 +324,49 @@ List all environments you have access to
 catalyze environments
 ```
 
+## <a id="files"></a> files
+
+The `files` command gives access to manage service files for your environment. The files command can not be run directly but has sub commands.
```
Comment (nateradtke) [inline Docs.md]: Reword to indicate "list and view" not "manage" service files. Also we should include a description about what service files are. They are the configuration files/settings for the application running inside of Catalyze containers (examples may include nginx site configs, logstash settings, etc).

File: commands/files.go [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+			file = helpers.RetrieveServiceFile(service.ID, f.ID, settings)
+			break
+		}
+	}
+	if file == nil {
+		fmt.Printf("File with name %s does not exist. Try listing files again by running \"catalyze files list\"\n", fileName)
+		os.Exit(1)
+	}
+	filePerms, err := strconv.ParseUint(file.Mode, 8, 32)
+	if err != nil {
```
Comment (nateradtke) [inline commands/files.go]: Default permissions to 0644

**Comment by nateradtke** [comment]
Looks pretty good. Built locally and exercised commands

**Comment by jault3** [comment]
The last two commits 099b8d10ff4f282d9c404078cd63762b3a3fc151 and 9f650425f0172752204d61e5f55de4d7845d324a fix issue #18. The git commands on association no longer run with `sh -c "..."` and instead use the git commands directly. Also the username and password prompt has been fixed for the default Command Prompt only.

Fixes #18

**Comment by jault3** [comment]
Fixes #17

**Comment by jault3** [comment]
Fixes #19

**Pull request merged**